langchain/src/vectorstores/tests/vectara.int.test.ts

```diff
@@ -94,7 +89,7 @@ describe("VectaraStore", () => {
       apiKey: process.env.VECTARA_API_KEY || "",
     };
 
-    test.skip("with fakeEmbeddings doesn't throw error", () => {
+    test("with fakeEmbeddings doesn't throw error", () => {
```

Comment:
Can you leave the .skip in? We don't run every integration test every time, they're for testing on an as-needed basis.